src/node_contextify.h

```diff
@@ -0,0 +1,100 @@
+#include "node_internals.h"
+#include "node_watchdog.h"
+#include "base_object-inl.h"
+
+namespace node {
+
+using v8::Array;
+using v8::Boolean;
+using v8::Context;
+using v8::FunctionCallbackInfo;
+using v8::Local;
+using v8::Name;
+using v8::Object;
+using v8::Persistent;
+using v8::PropertyCallbackInfo;
+using v8::PropertyDescriptor;
+using v8::Value;
+using v8::WeakCallbackInfo;
+
+class ContextifyContext {
```

Comment:
Put this class in a new namespace under `node`, like `node::contextify::ContextifyContext`.